docs/reference/objects/product/index.md

```diff
@@ -65,7 +65,7 @@ timestamp
 {: .label .fs-1 }
 
 - If the product has a fixed date this returns the start date of the event as a timestamp, this can then be used in conjunction with Liquid's [built-in filters](https://shopify.github.io/liquid/filters/date/).
-- If the product is an experience with multiple dates, this returns the start date of the next upcoming or ongoing date if there is one, or the most recent date if all dates are in the past.
+- If the product is an experience with multiple slots, this returns the start date of the next upcoming or ongoing date if there is one, or the most recent date if all slots are in the past.
```

Comment:
**question** Should this say: `this returns the start date of the next upcoming or ongoing slot if there is one`? as `the start date of the next upcoming or ongoing date` sounds a bit weird 🤔

Reply:
Yes can definitely be more clear here, it now reads:
`If the product is an experience with multiple slots, this returns the start date of the next upcoming or ongoing slot if there is one, or the start date of the most recent slot if all slots are in the past.`
Verbose, but accurate.

Reply:
I think that's much better, even if a bit longer 🚀